LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -264,7 +264,10 @@ extension TherapySettingsView {
 
     private var basalRatesSection: Card {
         card(for: .basalRate) {
-            if let items = viewModel.therapySettings.basalRateSchedule?.items, let supportedBasalRates = viewModel.pumpSupportedIncrements()?.basalRates {
+            if let items = viewModel.therapySettings.basalRateSchedule?.items,
+               let supportedBasalRates = viewModel.pumpSupportedIncrements()?.basalRates,
+               let total = viewModel.therapySettings.basalRateSchedule?.total()  {
+                Text(LocalizedString("\(round(total * 100) / 100.0) U/day", comment: "Scheduled Basal Daily Total"))
```

Comment:
This needs to be a format string, for localization. See other examples where String(format...) is used